docs/application/tizen-studio/release-notes/2-5-release-notes.md

```diff
@@ -0,0 +1,82 @@
+# Tizen Studio 2.5 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   SDB
+	-  wait-for-device command added, when this command is issued sdb waits indefinitely for the target to come to 'device' state.
+-   RT IDE Certificate Manager
+	-   Integrated Certificate Manager into RT IDE
+-   Baseline SDK Size Optimizations
+	-   Optimized Baseline SDK size from 480MB to 320MB
+
+### Fixed Bugs
+-   Certificate Manager
+	-   Added validation of .pem & .der files while importing as a new IoT profile in Certificate Manager
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
+	-   Fixed a issue of target directory not being able to set after invalid path.
+-   Base IDE
+	-   Supporting "Show in History" view of eclipse team service. 
+-   CLI
+	-   Fixed CLI Build failure for IoT Headless App.
+	> **Note**
+	> If you have projects with Old CLI Build Scripts you need to re-create the build script by running "Export to CLI" before running CLI commands.
+-   Others
+	-   Fixed security issues in Tizen Studio Source code.
+
+### Known Issues
+-	Tizen Studio
+	-	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+	-	In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+-	Common
+	-   If you install the Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+-	Web and Native IDE
+	-   From Tizen Studio 2.0 onwards, the Connection Explorer is replaced with the Device Manager, this can cause errors in the Connection Explorer view. You can fix this in two ways:
+	    -   Reset the perspective.
+            In the Tizen Studio menu, select Window > Perspective > Reset Perspective
+	    -   After updating to the Tizen Studio 2.0, run the *eclipse.exe -clean -clearPersistedState* command. Then launch the Tizen Studio normally.
+	-   You can create unit tests for Tizen 2.3.2 and higher version projects only. Now the Tizen Studio does not support unit testing for older versions.
```

Comment:
You can create unit tests for Tizen 2.3.2 and later version projects only. Now the Tizen Studio does not support unit testing for earlier versions.